src/core/resources/test-reader.cfg

```diff
@@ -0,0 +1,6 @@
+sensor:
```

Comment:
czy to jest cały config sensora(to co on wczytuje na start, nie to co dostaje po requestowaniu z cc)?
generalnie potrzeba zeby w nim bylo id sensora, jego typ, oraz domyslny adres+port CC na ktory posle request po adres centralki etc..